tubular/dates.py

```diff
@@ -429,14 +433,15 @@ class DateDifferenceTransformer(BaseDateTwoColumnTransformer):
         Name given to calculated datediff column. If None then {column_upper}_{column_lower}_datediff_{units}
         will be used.
     units : str, default = 'D'
-        Numpy datetime units, accepted values are 'D', 'h', 'm', 's'
+        Accepted values are "week", "fortnight", "lunar_month", "common_year", "custom_days", 'D', 'h', 'm', 's'
```

Comment:
Q: Not sure I understand the intent here, why are we expanding functionality at the same time as deprecating the transformer?

Reply:
my bad - misread the PR due to some collapsed lines!